src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -41,6 +42,7 @@ public class DocumentBuilder : IDisposable
         public DocumentBuilder(
             IEnumerable<Assembly> assemblies,
             ImmutableArray<string> postProcessorNames,
+            string templateHash,
```

Comment:
templateHash is a little different from intermediateFolder, the latter is from user's input  so it could be null while templateHash shouldn't be null for non-test usage.

---
In reply to: [75430563](https://github.com/dotnet/docfx/pull/621#discussion_r75430563) [](ancestors = 75430563)